setup.py

```diff
@@ -0,0 +1,39 @@
+"""Setup for diagnostic-feedback XBlock."""
+
+import os
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='xblock-diagnostic-feedback',
+    version='0.1',
```

Comment:
This is not the version you are trying to install into edx/edx-platform.  Can you update this branch with the right code?